src/Stratis.Bitcoin.IntegrationTests/Wallet/SendingTransactionOverPolicyByteLimitSpecification_Steps.cs

```diff
@@ -0,0 +1,119 @@
+            this.nodeGroupBuilder = new NodeGroupBuilder(this.CurrentTest.DisplayName);
+        }
+
+        protected override void AfterTest()
+        {
+            this.nodeGroupBuilder.Dispose();
+        }
+
+        private void two_connected_nodes()
+        {
+            this.nodes = this.nodeGroupBuilder
+                .StratisPowNode(NodeOne).Start().NotInIBD().WithWallet(WalletName, WalletPassword)
+                .StratisPowNode(NodeTwo).Start().NotInIBD().WithWallet(WalletName, WalletPassword)
+                .WithConnections()
+                    .Connect(NodeOne, NodeTwo)
+                    .AndNoMoreConnections()
+                .Build();
+        }
+
+        private void node1_builds_undersize_tx_to_send_to_node2()
+        {
+            Node1BuildsTransactionToSendToNode2(UnderPolicyByteLimitUnspentTransactionOutputs);
+        }
+
+        private void node1_builds_oversize_tx_to_send_to_node2()
+        {
+            Node1BuildsTransactionToSendToNode2(OverPolicyByteLimitUnspentTransactionOutputs);
+        }
+
+        private void Node1BuildsTransactionToSendToNode2(int txoutputs)
+        { 
+            this.CoinBaseMaturity = (int) this.nodes[NodeOne].FullNode.Network.Consensus.Option<PowConsensusOptions>().CoinbaseMaturity;
+
+            this.MineBlocks(this.nodes[NodeOne]);
+
+            var nodeTwoAddresses = this.nodes [NodeTwo].FullNode.WalletManager().GetUnusedAddresses(new WalletAccountReference(WalletName, WalletAccountName), txoutputs);
+
+            var nodeTwoRecipients = nodeTwoAddresses.Select(address => new Recipient
+            {
+                ScriptPubKey = address.ScriptPubKey,
+                Amount = Money.COIN
+                }).ToList();
+
+            this.transactionBuildContext = SharedSteps.CreateTransactionBuildContext(WalletName, WalletAccountName, WalletPassword, nodeTwoRecipients, FeeType.Medium, 101);
+
+            try
+            {
+                this.transaction = this.nodes[NodeOne].FullNode.WalletTransactionHandler().BuildTransaction(this.transactionBuildContext);
+                this.transactionFee = this.nodes[NodeOne].GetFee(this.transactionBuildContext);
+                this.nodes[NodeOne].FullNode.NodeService<WalletController>().SendTransaction(new SendTransactionRequest(this.transaction.ToHex()));
+            }
+            catch (Exception e)
+            {
+                this.caughtException = e;
+            }
+        }
+
+        private void transaction_is_just_under_100KB_limit()
+        {
+            this.transaction.GetSerializedSize().Should().BeInRange(99000, 100000);
+        }
+
+        private void node1_fails_with_oversize_tx_wallet_error()
+        {
+            this.caughtException.Should().BeOfType<WalletException>().Which.Message.Should().Contain("Transaction's size is too high");
+        }
+
+        private void node1_succeeds_sending_tx_to_node2()
+        {
+            this.caughtException.Should().BeNull();
```

Comment:
you can actually check it makes it to the other node beyond just checking not null.  You can check the mempool on the other node or the block if you are mining after the transaction send

Reply:
Oversize transactions fail in the wallet prior to send due to NBitcoin policy constraints (StandardTransactionPolicy::MaxTransactionSize = 100000)

Reply:
Yes but this one is used in the success case right ?